### ISSUE TITLE
(very) minor issue in CoC reporting guide re phone numbers

### ISSUE THREAD

**Issue by swaldman3** [carpentries/policies#7]
The CoC reporting guide (https://software-carpentry.org/CoC-reporting/) says,

> You can also contact Carpentry executive staff by telephone at 1-765-532-1126 (Jonah Duckles - Software Carpentry) or 1-530-341-3230 (Tracy Teal - Data Carpentry).

This assumes that the reader is in the USA.
I'm not sure whether we should change this be international numbers (+1....), or just insert "from the USA" or some such. It may depend on Jonah and Tracy's feelings about phone calls and time zones... ;-)

**Comment by jduckles** [comment]
Yeah, I think we can provide some TZ guidance here. I'm actually in NZ, so trying one number then the other actually provides some amount of TZ coverage.
